kibana-reports/public/components/main/report_details/report_details.tsx

```diff
@@ -119,19 +119,18 @@ export function ReportDetails(props) {
   };
 
   const parseTimePeriod = (queryUrl: string) => {
-    let timeString = queryUrl.substring(
-      queryUrl.lastIndexOf('time:'),
-      queryUrl.indexOf('))')
-    );
+    let timeStringRegex = /time:\(from:.*,to:.*\)/;
+
+    let timeStringRegexArray = timeStringRegex.exec(queryUrl);
+    let timeString = timeStringRegexArray[0];
```

Comment:
just a note, you can probably do `const [timeString, fromDateString, toDateString] = queryUrl.match(/time:\(from:(.+),to:(.+?)\)/)` might be easier

also I think the lazy match in `to:.*?\)` is necessary? wouldn't `to:.*\)` match to the last `)`?

Reply:
![image](https://user-images.githubusercontent.com/32652829/99607886-9f1f7d00-29c1-11eb-9251-8f869f96a4ea.png)

Reply:
Good catch - will make change